libs/langgraph/tests/test_pregel_async.py

```diff
@@ -4980,7 +4982,7 @@ async def decider(data: State) -> str:
 
     app = workflow.compile()
 
-    if SHOULD_CHECK_SNAPSHOTS:
+    if SHOULD_CHECK_SNAPSHOTS and checkpointer_name == "memory":
```

Comment:
Why does this have to change?

Reply:
Because it's pointless to snapshot the same thing 10 times for different checkpointers